src/explore-education-statistics-frontend/src/components/__tests__/Feedback.test.tsx

```diff
@@ -0,0 +1,133 @@
+import Feedback from '@frontend/components/Feedback';
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import _feedbackService from '@frontend/services/feedbackService';
+import { FeedbackRequest } from '@common/services/types/feedback';
+
+jest.mock('@frontend/services/feedbackService');
+
+const feedbackService = jest.mocked(_feedbackService);
+
+describe('Feedback', () => {
+  test('renders initial state correctly', () => {
+    render(<Feedback />);
+    expect(screen.getByText('Is this page useful?')).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Yes this page is useful' }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'No this page is not useful' }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Report a problem with this page' }),
+    ).toBeInTheDocument();
+  });
+
+  test('renders correctly and submits when "Yes" is selected', async () => {
+    render(<Feedback />);
+    await userEvent.click(
```

Comment:
Although you'll currently see many places in the code where we're using the `userEvent` module directly, this is no longer the correct approach as Testing Library was updated in v14 with a new API that looks like:

```tsx
const user = userEvent.setup();

render(<Feedback />);

await user.click(...);
```

See: https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent

At least for new code we should be using the new API. For older usages of `userEvent`, we should try remember to update these to the new API when we encounter them.

Similar can be done below in rest of test case as well.